src/server/server.ts

```diff
@@ -181,6 +181,7 @@ app.get('/login/success', async (req: Request, res: Response) => {
     res.cookie('feide_auth', JSON.stringify(feideCookie), {
       expires: new Date(feideCookie.ndla_expires_at),
       encode: String,
+      domain: `.${config.feideDomain}`,
```

Comment:
Poenget her er å legge på en . foran domenet for å at subdomener skal plukke den opp. Det gjør at ai.ndla.no kan sjå at du er logga på ndla.no og dermed gi deg tilgang.